### ISSUE TITLE
BUG: Monk unarmored movement dont increase after 6th level

### ISSUE THREAD

**Issue by lrclucas** [foundryvtt/dnd5e#3505]
I have done the test with SRD starter hero Perrin:
It starts with 25ft movement at level 1 (25tf of movement)
![image](https://github.com/foundryvtt/dnd5e/assets/151626786/76119431-7b34-4007-8bdf-74ce4a5b15b9)

Then at 2nd level it gains 10ft (which works -> goes to 35)
![image](https://github.com/foundryvtt/dnd5e/assets/151626786/8a021613-d701-49c4-bf6b-aca30fbc81be)
![image](https://github.com/foundryvtt/dnd5e/assets/151626786/c3f65371-de9e-4444-b71a-88e0a1d58a80)

But then, at 6th level, it should increase the bonus to 15ft (going to 40), but it is not applied
![image](https://github.com/foundryvtt/dnd5e/assets/151626786/236cbe86-d856-43de-8284-d3de2112edbc)
![image](https://github.com/foundryvtt/dnd5e/assets/151626786/cf91e96b-b491-4287-bf1b-37ac4f510d6c)

I am leveling up with these steps:

1. Going to features tab
2. click edit on top left
3. select desired level in class dropdown

**Comment by maxobremer** [comment]
I don't think this is possible without a module like DAE as at the moment the effect for unarmed movevement is just +10 to walking speed and I think you need DAE to be able to use @scale.monk.unarmored-movement as an effect value in an active effect.
As such its less of a bug and more of a system limitation

**Comment by krbz999** [comment]
Related issue: https://github.com/foundryvtt/dnd5e/issues/3060

**Comment by lrclucas** [comment]
So, if I replace the 10 with @scale.monk.unarmored-movement in the effect, it should work for all levels then?

**Comment by JPMeehan** [comment]
> So, if I replace the 10 with @scale.monk.unarmored-movement in the effect, it should work for all levels then?

the issue is that kind of @ value *isn't* parsed by the system currently.

**Comment by ddipaolo** [comment]
> > So, if I replace the 10 with @scale.monk.unarmored-movement in the effect, it should work for all levels then?
> 
> the issue is that kind of @ value _isn't_ parsed by the system currently.

It looks like the @value parsing is specific to "system.attributes.movement.walk".  If I add a separate effect on the Unarmored Movement feature that reads:

Attribute Key:  system.attributes.init.bonus
Change Mode:  Add
Effect Value:  @system.monk.unarmored-movement

Then I see my initiative value scaled appropriately as I level up.  It also works for other values (I tried "system.attributes.ac.bonus" as well), just not the movement speed for some reason.

**Comment by krbz999** [comment]
That is because `attributes.init.bonus` _does_ accept roll data, but the issue here is about using roll data for modifying movement, not using the value of movement as roll data elsewhere.

You can find all these properties on the wiki pages for roll data and active effects.

**Comment by Fyorl** [comment]
More appropriate to keep this open and marked as blocked actually
